BaseLib/LogogCustomCout.h

```diff
@@ -25,21 +27,25 @@ namespace BaseLib
 class LogogCustomCout : public logog::Target
 {
 public:
-#ifdef USE_MPI
+#if defined(USE_MPI) || defined(USE_PETSC)
 	/**
 	 * Constructor when MPI is involved
 	 *
 	 * @param all_rank_output_level  Minimum level to output messages from all MPI processes
 	 * @param mpi_comm               MPI communicator
 	 */
+#if defined(USE_PETSC)
+	LogogCustomCout(LOGOG_LEVEL_TYPE all_rank_output_level = LOGOG_LEVEL_INFO, MPI_Comm mpi_comm = PETSC_COMM_WORLD)
```

Comment:
So the maximum output level will be "Info"? What about debug output in debug builds?